### PR TITLE
Initial installation overview commit

### PR DESCRIPTION
**Pull request by brianhlin**
First step in the doc migration

### CODE REVIEW

File: docs/install-overview.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+|:---------------------|:------------------------------------|:---------------------------------------------------------|
+| Compute Element (CE) | `osg-ce.example.edu`                | OSG CE, HTCondor Central Manager, etc. (`osg-ce-condor`) |
+| Worker Nodes         | `w<em>NNN</em>.cluster.example.edu` | OSG worker node client (`osg-wn-client`)                 |
+
+Prepare the Batch System
+------------------------
+
+The assumption is that you have an existing batch system at your site. Currently, we support [HTCondor](http://research.cs.wisc.edu/htcondor/), [LSF](http://www-03.ibm.com/systems/platformcomputing/products/lsf/), [PBS](http://www.pbsworks.com) and [TORQUE](http://www.adaptivecomputing.com/products/open-source/torque/), [SGE](http://en.wikipedia.org/wiki/Oracle_Grid_Engine), and [Slurm](http://slurm.schedmd.com) batch systems.
+
+The easiest way to add a site to OSG is to install the OSG Compute Element (CE) on the central host of your batch system. In any case, before proceeding be sure that you can submit and successfully run a job from your OSG CE host into your batch system.
```
Comment (djw8605) [inline docs/install-overview.md]: I hope no one actually takes this advice.

Should we mention hosted ce's?

File: docs/install-overview.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Add OSG Software
+----------------
+
+If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.
+
+<span class="twiki-macro NOTE"></span> For sites with more than a trivial number of hosts, it is recommended to use some sort of configuration management tool to install, configure, and maintain your site. While beyond the scope of OSG’s documentation to explain how to select and use such a system, some popular configuration management tools are [Puppet](http://puppetlabs.com), [Chef](https://www.chef.io), [Ansible](https://www.ansible.com), and [CFEngine](http://cfengine.com).
```
Comment (djw8605) [inline docs/install-overview.md]: What is this NOTE?  I think we can have:

    !!! note
        For sites with more than a trivial...

This is the extension: https://pythonhosted.org/Markdown/extensions/admonition.html

File: docs/install-overview.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+Start GlideinWMS Pilot Submissions
+----------------------------------
+
+To begin running [GlideinWMS](http://www.uscms.org/SoftwareComputing/Grid/WMS/glideinWMS/) pilot jobs at your site, e-mail <osg-gfactory-support@physics.ucsd.edu> and tell them that you want to start accepting Glideins. Please provide them with the following information:
+
+-   The type of CE (HTCondor-CE or the now-unsupported GRAM-CE)
+-   The fully qualified hostname of the CE
+-   Resource/WLCG name
+-   OS major version of your worker nodes — EL 6, EL 7, or a mix of both?
+-   Do your worker nodes use gLExec?
```
Comment (djw8605) [inline docs/install-overview.md]: Do we support glExec anymore?

File: docs/install-overview.md (hunk 1 above)
Comment (brianhlin) [inline docs/install-overview.md]: Can we open this as an issue? I'd like to get the ball rolling on the transition and we can always fix specific prose later.

File: docs/install-overview.md (hunk 3 above)
Comment (brianhlin) [inline docs/install-overview.md]: It's still part of 3.3 so yes

Comment (djw8605) [inline docs/install-overview.md]: Yes it is on my radar...

File: docs/install-overview.md (hunk 1 above)
Comment (bbockelm) [inline docs/install-overview.md]: Yeah, let's establish the beachhead first, then clean up more.  I would prefer small tuneups that we see (i.e., "easiest" -> "most common") but let's not impede progress too much.

File: docs/install-overview.md (hunk 2 above)
Comment (bbockelm) [inline docs/install-overview.md]: Yup, please convert this to an admonition.

File: docs/install-overview.md (hunk 3 above)
Comment (bbockelm) [inline docs/install-overview.md]: For new sites / entries, we shouldn't mention `glexec`.  If it's not too much bother, just ditch this line.  We should keep the `glexec` documentation itself for OSG 3.3 users.

File: docs/install-overview.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+-   Do you accept multicore jobs?
+-   Maximum job walltime
+-   Maximum job memory usage
+
+Once the factory team has enough information, they will start submitting pilots from the test factory to your CE. Initially, this will be one pilot at a time but once the factory verifies that pilot jobs are running successfully, that number will be ramped up to 10, then 100.
+
+Verify Reporting and Monitoring
+-------------------------------
+
+To verify that your site is correctly reporting to the OSG, check [http://gratiaweb.grid.iu.edu/gratia/site?facility=%RED%&lt;SITE](http://gratiaweb.grid.iu.edu/gratia/site?facility=%RED%%3CSITE) NAME&gt;<span class="twiki-macro ENDCOLOR"></span> for records of your site reports. If you have enabled the OSG VO, you can also check <http://osg-flock.grid.iu.edu/monitoring/condor/sites/all_1day.html>.
```
Comment (bbockelm) [inline docs/install-overview.md]: GratiaWeb link is likely broken.  Switch to GRACC?

File: docs/install-overview.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+==============================
+
+This document outlines the overall installation process for an OSG site and provides many links into detailed installation, configuration, troubleshooting, and similar pages. If you do not see software-related technical documentation listed here, try the search bar to the left or contacting us at [goc@opensciencegrid.org](mailto:goc@opensciencegrid.org).
+
+Plan the Site
+-------------
+
+If you have not done so already, [plan the overall architecture of your OSG site](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SitePlanning). It is recommended that your plan be sufficiently detailed to include the OSG hosts that are needed and the main software components for each host. Be sure to consider [the operating systems that OSG supports](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SupportedOperatingSystems). A simple way to organize this information is in a table; for example, a basic site might include:
+
+<span class="twiki-macro TABLE" sort="off"></span>
```
Comment (bbockelm) [inline docs/install-overview.md]: Scrub out this line?

File: docs/install-overview.md (hunk 4 above)
Comment (brianhlin) [inline docs/install-overview.md]: @djw8605 @edquist What's the GRACC equivalent here?

Comment (djw8605) [inline docs/install-overview.md]: Probably say something like:

> You should be able to find your site name in the dropdown box here: https://gracc.opensciencegrid.org/dashboard/db/site-summary under the "site" filter.

**Comment by bbockelm** [comment]
@djw8605 - I tweaked a few of the remaining items.  Can you re-review and maybe approve things now?  Note that @brianhlin has enabled direct edits to his branch by maintainers -- might be easier for nitpicks.

**Comment by bbockelm** [comment]
(also, @brianhlin - you may want to fork a new branch for a PR if you don't want folks editing your `master`).

**Comment by brianhlin** [comment]
It's fine for the time being, I'll keep that in mind for the future, though.

**Pull request merged**